src/fides/api/util/tcf/tc_model.py

```diff
@@ -188,6 +191,12 @@ class TCModel(FidesSchema):
 
     num_pub_restrictions: int = 0  # Hardcoded here for now
 
+    ac_vendor_consents: List[int] = Field(
+        default=[],
+        description="A list of Google's Additional Consent Vendors for which the customer has opted in. These "
+        "are consented Google Ad Tech Providers that are not registered with IAB",
+    )
```

Comment:
The TC Model is an intermediate structure that I use to convert the TCF Experience contents into a TC string.  I'm doing some prerequisite steps to get the data in the expected format, plus adding a lot of validation, similar to what the frontend library does to build strings. 

To parallel the other constructs in this model, I'm now adding a list of AC vendors here as well when doing this translation that I will later use to build the AC string.

I left the name as `TCModel` even though it now has an AC string section because I wanted it to stay similarly named to what the FE is using - FidesModel also feels way too vague.

Reply:
this makes sense. to play devil's advocate a bit, it could be something like `FidesStringModel` but even that feels vague. we probably need a better field name than `fides_string` (`fides_consent_string` ?!) but that ship has sailed and i'm not going to suggest we go back now.

anyway, i don't see any major issues having this in the `TCModel`, that doesn't feel too problematic!